lib/net/http_fetch_plugin.js

```diff
@@ -229,6 +229,15 @@ shaka.net.HttpFetchPlugin = class {
     }
     return !!(window.fetch && window.AbortController);
   }
+
+  /**
+   * Determine if progress event is supported.
```

Comment:
In these particular plugins, there is no runtime determination to be made.  So I think these methods are not needed.  You can just pass `true` to the corresponding argument of `registerScheme()`.

Reply:
ok!